ABACAS2/abacas2.nonparallel.sh

```diff
@@ -82,6 +82,8 @@ if [ -z "$ABA_SPLIT_PARTS" ] ; then
 fi
 if [ "$cores" -eq 0 ]; then
 	cores=$(nproc) # If cores=0, use all cores available by default
+elif [ "$cores" -eq 1 ]; then
+	ABA_LOW_MEM="yes"
```

Comment:
Is ABA_LOW_MEM being exported here as env variable (in way it occurs for if statement on lines 88-89)